Sming/Components/Partition/src/Arch/Esp8266/base.csv

```diff
@@ -0,0 +1,3 @@
+# Name,   Type, SubType, Offset, Size,    Flags
```

Comment:
Maybe it will be better to have this ARCH specific files under Sming/Arch/Esp8266/... . This way every architecture will have all arch specific configurations in one place.

Reply:
I think this whole approach needs clarifying and simplifying. I have some thoughts which I'll set out below.